fuzzing/tools/launcher.py

```diff
@@ -0,0 +1,26 @@
+# Lint as: python3
+"""
+This is the launcher script to provide a uniform command line interface 
+behind a number of arbitrary fuzzing engines.
+"""
+
+from absl import app
+from absl import flags
+import os
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_integer('timeout', 20, 'test timeout.', lower_bound=0)
```

Comment:
Flags should be documented more thoroughly. Here someone seeing `test timeout` will likely not understand what this is about. I'll give a suggestion here, and expect you'll apply the same principles for future flags:

```
The maximum duration, in seconds, of the fuzzer run launched.
```